docs/en/setup/service-agent/java-agent/README.md

```diff
@@ -105,6 +105,10 @@ property key | Description | Default |
 `profile.duration`|Max monitor segment time(minutes), if current segment monitor time out of limit, then stop it.|`10`|
 `profile.dump_max_stack_depth`|Max dump thread stack depth|`500`|
 `profile.snapshot_transport_buffer_size`|Snapshot transport to backend buffer size|`50`|
+`meter.active`|If true, skywalking agent will enable sending meters. Otherwise disable meter report.|`true`|
+`meter.report_interval`|Report meters interval|`20`|
+`meter.rate_counter_name`|Rate by agent counter meter names, spilt by ","|``|
```

Comment:
I would like to remove this, but using `counter.create().mode(RATE)` to replace.

Reply:
In MicroMeter, we could provide parameters to `SkywalkingMeterRegistry` to do similar thing.